server.py

```diff
@@ -432,6 +432,26 @@ def put(self, region, id):
             tournament.regions = args['regions']
 
         dao.update_tournament(tournament)
+
+    def delete(self, region, id):
+        """ Deletes a pending tournament.
+            If a (non-pending) tournament with this id exists, tell the user they can't do that.
+            Route restricted to admins for this region. """
+        dao = Dao(region, mongo_client=mongo_client)
+
+        user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(user, region):
+            return 'Permission denied', 403
```

Comment:
You should be checking that the user is an admin for the tournament's regions, not an admin for the region in the URL path.